sumologic/resource_sumologic_password_policy.go

```diff
@@ -0,0 +1,157 @@
+package sumologic
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceSumologicPasswordPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSumologicPasswordPolicyCreate,
+		Read:   resourceSumologicPasswordPolicyRead,
+		Update: resourceSumologicPasswordPolicyUpdate,
+		Delete: resourceSumologicPasswordPolicyDelete,
+
+		Schema: map[string]*schema.Schema{
+			"min_length": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  8,
```

Comment:
Let's define a top level variable (like `DefaultPasswordPolicy`) with default values and use them here and in the test code (`newDefaultPasswordPolicy`).